spec/requests/apartments_spec.rb

```diff
@@ -2,6 +2,26 @@
 
 RSpec.describe "Apartments", type: :request do
   describe "GET /index" do
-    pending "add some examples (or delete) #{__FILE__}"
+    it "gets a list of apartments" do
+      Apartment.create(
+        street: "123 other way",
+        city: "nashville",
+        state: "Tennessee",
+        manager: "Steve",
+        email: "apartmetnOther@gmail.com",
+        price: "2100",
+        bedrooms: 2,
+        bathrooms: 1,
+        pets: "no",
+        image:"https://furnishrcdn-134f8.kxcdn.com/wp-content/uploads/2020/02/Sandy_apartment_daylight-e1584735599779.jpg"
+      )
+
```

Comment:
It doesn't look like you created a user? Check out the README of this project for more info.